chart/base.js

```diff
@@ -639,7 +639,7 @@ var Chart = Backbone.Model.extend ({
 
 				let dataSheetName = "Table";
 
-				if (me.dataPerSheet) {
+				if (me.dataPerSheet && me.charts.length > 1) {
```

Comment:
don't do that' I want to have the second tab named as chart even if there is only one chart

Reply:
I have reverted this line. 
But be aware that in the case of using the 'dataPerSheet' property without multiple charts (except multiple charts export with one item in the array) will be a reference error in the excel file.